cg/apps/demultiplex/sbatch.py

```diff
@@ -28,15 +28,13 @@
     "dragen": """
 log "dragen --bcl-conversion-only true \
 --bcl-input-directory {run_dir} \
---output-directory {unaligned_dir} \
---bcl-sampleproject-subdirectories true \
```

Comment:
@karlnyr This is the change that might cause the current sample sheets to fail right? If we remove the argument we also need to remove the `Sample_Project` column in the sample sheet. Have I understood it correctly?

Reply:
Yes - but by simply using the new SampleSheet -  SampleSheetCreatorV2 - we will have a sample sheet which is compliant with the updated options we use.